test/factories/tasks.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :task do
+    name
+    description
+    author_id { 1 }
+    assignee_id { 1 }
```

Comment:
author_id и assignee_id должны ссылаться на соответствующие фабрики

Reply:
> author_id и assignee_id должны ссылаться на соответствующие фабрики

```suggestion
    assignee_id { 1 [}](url)
```Разве в задании "Генерация моделей" не сказано, что эти два параметра должны остаться статичными, до следующего задания?

Reply:
> author_id и assignee_id должны ссылаться на соответствующие фабрики

Разве в задании "Генерация моделей" не сказано, что эти два параметра должны остаться статичными, до следующего задания?

Reply:
Окей, отправь задание заново на проверку